### DISCUSSION TITLE
Cruiser Pro

### DISCUSSION THREAD

**Discussion by devemux86** [devemux86/cruiser#489] [General]
**Cruiser Pro** subscription currently offers the following large features:
- Voice navigation
- Different route profiles for sections
- No-go areas
- Record GPS tracks
- Many configuration options

Everything else is completely free and there are so many features in all application areas.

However, whether we like it or not, it's app subscriptions that can keep mobile apps alive. :slightly_smiling_face: 

Do you think we should move something more to Cruiser Pro to get more users to sign up?

**Comment by MotoUKRider** [comment]
If any feature (or limitation of a feature) is possible, then you could provide some restrictions under "Free", ie:

- Only 1 Favorite Group visible on map
- Limit nearby Favorite/POI somehow
- Offline Map 2 or 3 files only
- 2D View only

There's also your time - is it possible that the "Pro" features also create the most work?

For my 2c: Cruiser costs half as much as my previous app but I get more than 2x the value regardless of the support here, helping to develop map themes and finding the most obscure of bugs/routing-issues :vulcan_salute:

**Reply by devemux86** [reply]
> If any feature (or limitation of a feature) is possible

Both are possible.

**Reply by Leeb64** [reply]
A “free trial phase” of approximately 2 weeks.
Then only minimal functions...
But what would also be very important:
A description!
I know 3 cruiser users myself and regularly tutor them.
I can imagine it (and you can see it in the reviews (the ones with 1 star))... people don't understand it.

**Reply by bios4** [reply]
I could imagine to make the following (large) features "Pro only" in addition:

- My collection (or at least limit it to "X" groups with max. "Y" entries each in the free version as already suggested)
- Offline maps

The issue with the (missing) manual is a valid downside...

Should we (forum users) try to start to create one, even when it is only step-by-step?

**Reply by Leeb64** [reply]
Imagine you are looking for a navigation APP.
If you don't understand anything from the start, then just take the next APP that you might understand better - logically!
At least the basic principle with the various route calculation options, how the BRouter works, map downloads... just basic things like that... to make it easier to get started...

**Comment by DolomitesTom** [comment]
I would limit the free version more, to a minimum, no offline maps, no download manager, no navigation, no routing, no options for planning, maybe pure follow mode. If possible a free trial period of 4 weeks with all functions.

10 euros per year are not too much for such good pro functions.

**Reply by Leeb64** [reply]
But you also have to hook people somehow...

**Reply by Leeb64** [reply]
It should also be emphasized that the APP can work completely offline.

**Comment by devemux86** [comment]
Another idea could be the "Extended settings" switch (i.e. app great customization) to be a Pro feature.

**Comment by DIrgi** [comment]
### Using a desktop surface for planning tool, connected to cruiser?
(like kurviger, calimoto et al)

> _I know that you did the former programming or at lest participate in the kurviger app (afaik up to 2.0). No matter if you still do so or what were the reasons why (if so) you don't do this anymore. However, after all I used and still use kurviger as my main planning tool on the Desktop. I never used the kurviger 2.0 app because it was not available for iphone at these days and next by I used a Garmin navi so the app was obsolet anyway. 
> However meanwhile I have an android tablet and I want to replace my medieval Garmin Navi. In comparison I liked cruiser more then the kurviger app._

Never the less, what I really like in kurviger is the planning. So do you see a chance to have something similar available, connected to/with the app (and like the other) only for plus users? - I know that kurviger format can be used on cruiser but it's not directly connected.
I assume the don't allow you a connection to kurviger otherwise it won't make sense the have two navigation apps.

**Reply by devemux86** [reply]
BRouter (offline) currently works on Android. It requires work to be used on desktop.

The best alternative is [BRouter website](https://brouter.de/brouter-web/) using the Cruiser profile:

- https://github.com/devemux86/cruiser/discussions/272

(however it does not support different options per route section like Cruiser)

**Reply by DIrgi** [reply]
Not bad 👍
I wasn't aware about the existence so far. I played a little bit around but in direct comparison I like kurviger desktop more and finally there's not really a difference it's as well not connected with cruiser.
But good to know and thank you for the link

**Comment by Cer-o** [comment]
Personalmente dejaría que quién se instale la aplicación la pueda probar por 7 ó 10 días y pasado ese tiempo que pase por caja si quiere seguir utilizándola.
Crear, mantener y evolucionar una aplicación conlleva un gran trabajo y debe ser remunerado.

**Comment by airald** [comment]
Software should be always paid to users, but sometimes give some open features makes more users will buy it in the future. It's complicated found the balance.

IMO, offline maps, and all "extended settings".

The ability to let users to try it free is always a good option. I know that you can buy one app and then uninstall and give the money back, but mos of the people will don't do it. When the limit is time from installation, it's possible that you install and when you go to try it's out of date. 

For me, the best option is have a free version that allow user to try the app and see what really can do. And for certain pro options like navigate, limit it to certain number of km. Users can investigate the app, try at home and then try on a simple route after 1 week or one month.

Another option (I'm don't know if it's possible or easy) should be the option to start the routing trial period when user wants.

There are a lot of different ways to do, but my oppinion is that free version should be more limited.

**Comment by umpfmpf** [comment]
I think the fee for a year is perfectly fine for the app to offer so much! 
You still need an old smartphone, but a good navigation system also costs a few hundreds euro today. And then offers far fewer gadgets! And who has been driving his navigation system for more than 6-8 years? 
The bottom line is that the solution with cruiser and smartphone is definitely profitable!

Other apps offer e.g. the possibility to test for 4 weeks, after that there is only purchase, but the app no longer works. 

For me a full version test for 2 weeks and then a very limited version will stimulate the purchase the most.

**Comment by devemux86** [comment]
Set different profiles in individual route sections could also be a Pro feature?

And normal operation uses the global route profile in all route sections.

**Reply by airald** [reply]
YES, IMHO, should be a Pro feature

**Reply by bios4** [reply]
I think the following features could also be Pro-only:

- https://github.com/devemux86/cruiser/discussions/254
- https://github.com/devemux86/cruiser/discussions/449
- https://github.com/devemux86/cruiser/discussions/499
- https://github.com/devemux86/cruiser/discussions/502

**Reply by devemux86** [reply]
@bios4 thanks for the suggestion.

- https://github.com/devemux86/cruiser/discussions/254

Section profiles moved to Pro only, route profile can be set to anything.

- https://github.com/devemux86/cruiser/discussions/502

Section management, like recalculate or delete a section has also been moved to Pro.

**Comment by devemux86** [comment]
There may be new subscription prices (for new users):
- **3 €** / month
-  **24 €** / year

Subscription price changes only apply to new purchases.
Existing subscribers are not affected by the price change.

---

Your thoughts?

**Reply by airald** [reply]
Absolutely understandable.

Also we need to talk about the existing subscriptions. How many years should mantain the initial price? Inflation exists!

**Reply by devemux86** [reply]
Existing subscriptions can also be updated with the new prices, if we want:
- An opt-in message is displayed by Google Play to users 30 days before the next renewal.
- They can accept the price change or the subscription is canceled.

So this is also possible, as Google covers everything.

**Reply by kds112** [reply]
Well, first I would release v5 and look after an increasing number of subscribers before increasing the price by 140%...I think this step is too big...

**Reply by devemux86** [reply]
@kds112 

We also have to consider the many new features that were implemented in the last 2 years for free.

**Reply by Ciutnik** [reply]
@devemux86 
> Your thoughts?

I think you have already made up your mind 😉 I must admit that the scale of the increase is somewhat shocking - especially if it concerns subscriptions. 

Remember that for many motorcyclists/bikers the winter season is not attractive - they may drop their annual subscription in favour of a 4-6 month subscription and then you lose customers (and, on an annual basis, income)...

And perhaps you would like to offer a time-limited offer for Forum members, i.e. loyal users and dedicated testers, in the form of a one-off lifetime fee? Make it €50 or €80 - I wouldn't hesitate for a moment to take advantage 🙂 If you don't count it's an upfront fee for a 5-7 year subscription.

**Reply by devemux86** [reply]
@Ciutnik 

Comparing the prices with other apps, they have higher subscriptions.

One can argue about the features of each app,
but development cannot continue for free, one-time purchases or at low prices. :slightly_smiling_face:

**Comment by umpfmpf** [comment]
I think everyone should be paid properly for their work. 
Nobody has to get rich, but it should take a carefree life! 

And if I now calculate: 

I use a motorcycle navigation system for €300 for about 3-4 years, one for €500-600 for about 7 years. 
If I unplug a cell phone to use cruiser, I get around €100 for a small navigation system or €300 for a large navigation system. 
That's around €25-30 a year for the corresponding software during this time.

So the price for cruiser should be justified!
The most software offers far fewer customization options. 

And so far I'm still hoping to see the 3D navigation in V5 at some point.

**Reply by DolomitesTom** [reply]
> That's around €25-30 a year for the corresponding software during this time.

A navigation device still has some benefits in wet conditions, robustness and traffic data. Tomtom app costs 20€ a year including traffic data.

**Reply by WolfgangZ42** [reply]
> A navigation device still has some benefits in wet conditions, robustness and traffic data.

No - I don't see any advantages neither in wet conditions nor in robustness and I don't need traffic data when riding my motorcycle.

**Comment by 0709wiwiwi** [comment]
Don't contaminate github. Discuss technology.

**Reply by airald** [reply]
Unfortunately money is necessary to develop technology...

**Comment by abdullahO2** [comment]
I strongly support the proposed price adjustments. Let's remember the immense amount of free development @devemux86 has contributed, particularly to the Mapsforge library. He made it open-source, allowing other developers to build upon it, often for profit. This generosity deserves recognition and support.

@devemux86's dedication is unparalleled. He's incredibly responsive to the community, consistently engaging with discussions and addressing issues. This level of support is rare and invaluable.

Charging for Pro features is entirely reasonable. These features enhance the experience but don't cripple the core functionality of the free version. If the price isn't agreeable, plenty of alternative apps exist. No one is forced to use Cruiser.

Furthermore, Cruiser offers incredible value even with the Pro features. Compare the annual cost to other navigation apps or dedicated GPS devices. Cruiser is significantly more affordable, especially considering its extensive feature set and customization options.

This price increase is essential for the long-term sustainability of the project. Developing and maintaining an app like Cruiser requires time, effort, and resources. A sustainable income allows @devemux86 to continue improving the app, adding new features, and providing excellent support.

Perhaps consider adding a "Thank You" tier at a higher price point for users who want to contribute more and show extra appreciation for your hard work.

Finally, I appreciate the transparency in explaining the rationale behind the price change. Open communication builds trust and strengthens the community. Thank you for all your hard work, @devemux86!
